anime_downloader/extractors/vidstream.py

```diff
@@ -1,42 +1,70 @@
 import logging
 import re
+import json
+
+from anime_downloader.config import Config
 from anime_downloader.extractors.base_extractor import BaseExtractor
+import anime_downloader.extractors as extractors
 from anime_downloader.sites import helpers
 
 logger = logging.getLogger(__name__)
 
 class VidStream(BaseExtractor):
     def _get_data(self):
-        QUALITIES = {
-        "360":[],
-        "480":[],
-        "720":[],
-        "1080":[],
+
+        links = {
```

Comment:
Are all these supported by the same logic? You could just subclass and add different classes, if thats the case

Reply:
It just links urls to extractors, I don't see any need for classes.